tools/shell-commands/src/main/resources/requestBody/2.2.0/base_index_mapping.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Could be rename to base_index_withRollover_request.json ?

Reply:
Done